docs/source/usage/workflows/ionization.rst

```diff
@@ -0,0 +1,187 @@
+                    3.874,
+                    /* 1s^2 */
+                    6.665,
+                    6.665);
+            } // namespace picongpu::ionization::effectiveNuclearCharge
+
+        .. note::
+            see `wikipedia <https://en.wikipedia.org/wiki/Effective_nuclear_charge>`_ for values or refer directly to the calculations by Slater or Clementi and Raimondi, [1]_ and [2]_.
+
+    .. attention::
+        In addition this file must also contain the Thomas-Fermi ionization parameters and cutoff settings.
+        We suggest to copy them from the default :code:`ionizer.param` and **not** adjust them.
+
+2. Define the ionization energies of all charge states of all ionization ion species of your simulation in :code:`ionizationEnergies.param`.
+
+    .. code-block:: c++
+
+        namespace picongpu::ionization::energies::AU
+        {
+            /* example Nitrogen */
+
+            /* ionization energy in eV */
+            PMACC_CONST_VECTOR(
+                float_X,
+                7, // number charge states
+                Nitrogen, // name you may reference this by later, remember to prepend the namespace and append _t!
+                14.53413 * UNITCONV_eV_to_AU,
+                29.60125 * UNITCONV_eV_to_AU,
+                47.4453 * UNITCONV_eV_to_AU,
+                77.4735 * UNITCONV_eV_to_AU,
+                97.89013 * UNITCONV_eV_to_AU,
+                552.06731 * UNITCONV_eV_to_AU,
+                667.04609 * UNITCONV_eV_to_AU);
+        }; // namespace picongpu::ionization::energies::AU
+
+    .. note::
+         see `NIST <http://physics.nist.gov/PhysRefData/ASD/ionEnergy.html>`_ for ionization energies of the elements [3]_
+
+
+3. Define mass and charge ratios of ions in the :code:`speciesDefinition.param`.
+
+    .. code-block:: c++
+
+        /* Example Nitrogen*/
+        namespace picongpu
+        {
+            /* mass and charge ratios with respect to electrons */
+            value_identifier(float_X, MassRatioNitrogen, 25514.325);
+            value_identifier(float_X, ChargeRatioNitrogen, -7.0);
+        } // namespace picongpu
+
+4. Define at least one electron species in :code:`speciesDefinition.param`.
+
+    .. code-block:: c++
+
+        /* Example electron Species */
+        namespace picongpu
+        {
+            using ParticleFlagsElectrons = MakeSeq_t<
+                particlePusher<UsedParticlePusher>,
+                shape<UsedParticleShape>,
+                interpolation<UsedField2Particle>,
+                current<UsedParticleCurrentSolver>,
+                massRatio<MassRatioElectrons>,
+                chargeRatio<ChargeRatioElectrons>>;
+
+            using ParticleAttributesElectrons = MakeSeq_t<position<position_pic>, momentum, weighting>;
+
+            using Electrons = Particles<PMACC_CSTRING("e"), ParticleFlagsElectrons, ParticleAttributesElectrons>;
+        } // namespace picongpu
+
+
+5. Define for each ion species a PIConGPU species
```

Comment:
This sentence makes no sense. What is a "PIConGPU species"?

Reply:
done